### PR TITLE
Fix UserSettings & ColorPicker

### PR DESCRIPTION
**Pull request by Zach777**

### CODE REVIEW

**Comment by Reousa** [comment]
Please rebase so I can review & merge.

File: Tree/Interface/MainMenu/OptionsOld/UserSettings.gd [hunk 1]
```diff
@@ -79,9 +88,11 @@ func _on_Gender_item_selected(ID : int) -> void:
...
-	Options.username = new_text
+	username = new_text
 	get_node(text_edit2).text = new_text
+	Log.trace(self, "_on_UsernameTextEdit_text_changed", "Change player's name.'")
 
 func _on_UsernameTextEdit2_text_changed(new_text : String) -> void:
-	Options.username = new_text
+	username = new_text
 	get_node(text_edit1).text = new_text
+	Log.trace(self, "_on_UsernameTextEdit2_text_changed", "Change player's name.'")
```
Comment (Reousa) [inline Tree/Interface/MainMenu/OptionsOld/UserSettings.gd:98]: Which one of these should be used, why are there two of the same func?

**Comment by Reousa** [comment]
Worth noting we'll still re-write this, later. 
For now we'll use it as is to get the new demo up and running.

File: Tree/Interface/MainMenu/OptionsOld/UserSettings.gd (hunk 1 above)
Comment (Zach777) [reply Tree/Interface/MainMenu/OptionsOld/UserSettings.gd:98]: They update the other's text field. Since there are two name input fields.

Comment (Reousa) [reply Tree/Interface/MainMenu/OptionsOld/UserSettings.gd:98]: I see, let's let it pass for now then, until it's rewritten.

Comment (Zach777) [reply Tree/Interface/MainMenu/OptionsOld/UserSettings.gd:98]: Worth noting that I can easily remove one of the text fields if wanted.

Comment (Reousa) [reply Tree/Interface/MainMenu/OptionsOld/UserSettings.gd:98]: It's more about the naming really, do so 👌

**Pull request merged**